app/furniture/journal/entry.rb

```diff
@@ -32,4 +32,12 @@ def to_html
   def to_param
     slug
   end
+
+  def location
+    journal.location + [self]
+  end
+
+  def self.model_name
+    @model_name ||= ActiveModel::Name.new(self, ::Journal)
+  end
```

Comment:
There's probably a  thing we could figure out to get rid of these; since it seems like something we're wanting to do quite regularly?